plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -388,8 +388,9 @@ static bool ffmpeg_hls_mux_start(void *data)
 	stream_key = obs_service_get_key(service);
 	dstr_copy(&path, path_str);
 	dstr_replace(&path, "{stream_key}", stream_key);
-	dstr_catf(&stream->muxer_settings, "http_user_agent=libobs/%s",
-		  OBS_VERSION);
+	if (dstr_is_empty(&stream->muxer_settings))
+		dstr_catf(&stream->muxer_settings, "http_user_agent=libobs/%s",
```

Comment:
I think it's better to use dstr_init to reinit muxer_settings and then concatenate the string.